java/src/main/java/com/google/cloud/dataproc/templates/kafka/README.md

```diff
@@ -125,4 +125,69 @@ bin/start.sh \
 --templateProperty kafka.bq.table=kafkaevents \
 --templateProperty kafka.bq.temp.gcs.bucket=templates-demo-kafkatobq \
 --templateProperty kafka.bq.await.termination.timeout=1200000
+```
+
+
+## 2. Kafka To GCS
+
+General Execution:
+
+```
+GCP_PROJECT=<gcp-project-id> \
+REGION=<region>  \
+SUBNET=<subnet>   \
+GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> \
+HISTORY_SERVER_CLUSTER=<history-server> \
+bin/start.sh \
+-- --template KAFKATOGCS \
+--templateProperty project.id=<gcp-project-id> \
+--templateProperty kafka.gcs.output.location=<gcs path> \
+--templateProperty kafka.bootstrap.servers=<kafka broker list> \
+--templateProperty kafka.topic=<kafka topic name> \
+```
+
+
+### Configurable Parameters
```

Comment:
`Configurable Parameters` should be removed.
As it is just a duplication of template.properties file

Reply:
Configuration parameters have been removed. Will push the changes and resolve conversation